website/docs/d/is_network_acl_rule.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
The docs are changed recently please see the latest docs and use that format please check this for refernce
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/docs/r/is_security_group_rule.html.markdown

format
description , link to cloud docs
Alphabetical order of arguments and attribute